xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererOMX.cpp

```diff
@@ -27,6 +27,7 @@
 #include "utils/GLUtils.h"
 #include "settings/MediaSettings.h"
 #include "windowing/WindowingFactory.h"
+#include "DVDCodecs/Video/DVDVideoCodecIMX.h"
```

Comment:
@FernetMenta I'm unable to find a file named "VideoCodecOpenMax.h" are you referring to "DVDVideoCodecOpenMax.h"?
I don't have an opinion about naming (other than I think it should be "Renderer"-something not "Render"-something, to match the naming of the other files https://github.com/xbmc/xbmc/tree/master/xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender). But should the decision be approved in any way?